airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/streams/base_insight_streams.py

```diff
@@ -60,6 +61,7 @@ def __init__(
         breakdowns: List[str] = None,
         action_breakdowns: List[str] = None,
         action_breakdowns_allow_empty: bool = False,
+        action_report_time: str = None,
```

Comment:
Instead of assigning None, we can set a default value since it will be reassigned in line 83 anyway.

Reply:
done :heavy_check_mark: